qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/BrokerLeaderChangeTest.java

```diff
@@ -82,6 +84,32 @@ public void shouldChangeLeaderAfterLeaderDies() {
     jobCompleter.close();
   }
 
+  @Test
+  public void shouldBeAbleToBecomeLeaderAgain() {
+    // given
+    final var firstLeaderInfo = clusteringRule.getLeaderForPartition(1);
+    final var firstLeaderNodeId = firstLeaderInfo.getNodeId();
+    assertThat(clusteringRule.getCurrentLeaderForPartition(1).getNodeId())
```

Comment:
Was that sometimes false?

Reply:
No was just to validate that it at the begining the first and then after the loop again - I had hope it makes the test more understanable :D but this failed :D